docs/reference/math/isInteger.md

```diff
@@ -0,0 +1,27 @@
+# isInteger
+
+Checks if `value` is an integer.
+
+## Signature
+
+```typescript
+function isInteger(value: any): boolean;
```

Comment:
```suggestion
function isInteger(value: unknown): value is number;
```

This function might server as a type predicate; we might add narrow the given type.